Makefile

```diff
@@ -159,6 +159,9 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 
 .PHONY: deploy
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	@if [ "$(IMG)" != "controller:latest" ];then \
```

Comment:
@vprashar2929 note for future use `[[` instead of `[` :) . Also why is this check needed? why not just cd and set ?

Reply:
I also had the same idea to do just cd but I noticed that since we already mantain the operator image with tag. It's best to have the flexibility to make setting the image optional. If anyone wants to use the latest image then can provide that in argument and if not it will pick up whatever image tag is present [here](https://github.com/sustainable-computing-io/kepler-operator/blob/c8e36ad8885e7705a2175756f51d75d97f969f33/config/manager/kustomization.yaml#L8).

Reply:
IMG = controller:latest isn't valid image so,  this will create a broken deployment anyway. I don't think `controller:latest` should be treated as somehow special. I will provide a PR that fixes this or please feel free to do that :)